src/utils/actions.js

```diff
@@ -342,9 +342,26 @@ const actions = [
           }
         },
     requiredInput: ['automation'],
+    featureFlag: 'ACTION_BUTTON_AUTOMATIONS',
+  },
+  {
+    value: 'refreshDynamicViews',
+    translation: 'Object.ActionButton.RefreshDynamicViews',
+    group: 'dynamicViews',
+    getActionCall:
+      ({ senseNavigation }) =>
+        async () => {
+          if (typeof senseNavigation.refreshDynamicViews === 'function') {
+            await senseNavigation.refreshDynamicViews();
+          }
+        },
+    requiredInput: [],
+    featureFlag: 'REFRESH_DYNAMIC_VIEWS_ODAG_POPUP',
   },
 ];
 
+export const getFFEnabledOrDefaultActions = (isEnabled) => actions.filter((a) => !a.featureFlag || isEnabled(a.featureFlag));
```

Comment:
This name is very confusing. Can it just be getActionsList or something?

Reply:
Now the name is changed.